packages/express-openapi/README.md

```diff
@@ -368,6 +368,57 @@ available in the affected operation handler method.
 * `'x-express-openapi-disable-validation-middleware': true` - Disables input
 validation middleware.
 
+#### Express Middleware to Validate Responses on All Routes
+
+By overriding the implementation of `res.send` within a middleware function, it is possible to perform `res.validateResponse` on all paths, and allow the paths to use `res.send` or `res.json`. The example below is how such a middleware might be added to `app.js`. For a TypeScript example see [TypeScript Example of Global Response Validation](#typescript-example-of-global-response-validation).
+
+```javascript
+function ValidateAllResponses(req, res, next) {
+    const strictValidation = req.apiDoc['x-express-openapi-validation-strict'] ? true : false;
+    if (typeof res.validateResponse === 'function') {
+        res.send = WrapSendFunction.bind(res, res.send, res.validateResponse);
```

Comment:
can you do something like this instead?

```
if (typeof res.validateResponse === 'function') {
  const originalSend = res.send;
  res.send = function expressOpenAPISend(...) // helps to see the function name when debugging

}
```

Reply:
I felt that with the function definition outside of the if-block and assignment of res.send that it might be more readable. Also I wasn't sure if res.json or another function would also need to be wrapped and reuse the logic. Is there a benefit to avoiding the use of bind?

I will update as you suggest. In testing, I didn't need to change res.json. And if someone has another need to wrap a separate function, they can refactor at that time.